### ISSUE TITLE
Kate - SOCIETIES WHERE THE RECORDING WAS MADE OUTSIDE THE SOCIETY'S HOMELAND

### ISSUE THREAD

**Issue by stellasilbert** [theglobaljukebox/cantometrics#30]
Kate's notes:
This is something that would be good to have a clearly stated policy on (and maybe it already exists!). I know Stella and I discussed it a couple of time. 
There seem to be two types of cases: 
A. Diaspora cultures: Cases where this is obvious from the society name (e.g., Scandinavian Americans, German Americans) -- here, the recording was made somewhere in the US, but the music etc. is presumably meant to represent the music of the homeland.  
  e.g., Song ID 9199 - German Americans (30136) - in the song metadata this society has a clearly distinguished local lat/lon vs. homeland lat/lon; in the culture metadata it has only a society lat/lon (which places it in the US).
B. Opportunistic/convenience recording at a location different from home location: Cases where the ethnomusicologist recorded one or more performers, who perhaps still lives in their 'home' culture at a different location. 
   e.g., Song ID 2994 - Pashtun (23497) - the song metadata notes: "Khyber Pakhtunkhwa Province, Pakistan [Recorded at Alan Lomax's apartment, New York, New York]"; only the homeland lat/lon is given in the song/culture metadata sheets
  e.g., [I can't find an example right now...] in some cases recordings were done in a regional hub (e.g., mid-sized town), and the lat/lon given for the song seem to be for the recording location as opposed to the homeland
In the short-term, the simplest solution might be to add a "homeland" lat lon to the culture metadata (not just the song metadata), and to fill that in when it differs from the recording lat lon AND is important to know (so, maybe not in the Pashtun example, but yes in the German American example)?

**Comment by stellasilbert** [comment]
When implementing the suggested solution (adding homeland lat lon to culture metadata where relevant), should I use the average homeland lat long from the song metadata? --Stella

**Comment by stellasilbert** [comment]
I added homeland lat and homeland lon columns on the society metadata sheet and filled them in for the societies that currently have songs with homeland lat-lons. Just for sake of time, I did not add homeland lat-lons for societies that are diasporic but don't have any songs in our datasets (like Finnish American and Scandinavian American). We can always add them later if needed but since they don't have any data on the GJB currently it didn't seem necessary.
